modules/aruco/samples/calibrate_camera_charuco.cpp

```diff
@@ -57,8 +57,8 @@ const char* about =
 const char* keys  =
         "{w        |       | Number of squares in X direction }"
         "{h        |       | Number of squares in Y direction }"
-        "{sl       |       | Square side lenght (in pixels) }"
-        "{ml       |       | Marker side lenght (in pixels) }"
+        "{sl       |       | Square side length (in meters) }"
+        "{ml       |       | Marker side length (in meters) }"
```

Comment:
Actually this is incorrect as far as I understand it. A calibrated setup always works on sizes in pixels of the camera coordinate system, not in real measurements. This is the same for a stereo setup calibration...